lib/server/idemix/idemixlib.go

```diff
@@ -37,7 +37,7 @@ func (i *libImpl) GetRand() (rand *amcl.RAND, err error) {
 	defer func() {
 		r := recover()
 		if r != nil {
-			err = errors.Errorf("failure: %s", r)
+			err = errors.Errorf("Failed to get rand: %s", r)
```

Comment:
Error strings should not start with a capital letter unless it is a proper noun or an acronym

Reply:
thanks for your review,I will  fix this it.